framework/base/View.php

```diff
@@ -162,6 +168,11 @@ public function render($view, $params = [], $context = null)
      */
     protected function findViewFile($view, $context = null)
     {
+        $cacheKey = get_class($context) . '@' . $view;
```

Comment:
1. `$context` can be `null`
2. `Yii::$app->getViewPath()` on lines [:181](https://github.com/yiisoft/yii2/pull/12578/files#diff-64dcde6a6056dd54d40f3d5a0a79b2efR181), [:190](https://github.com/yiisoft/yii2/pull/12578/files#diff-64dcde6a6056dd54d40f3d5a0a79b2efR190) could be changed during the runtime
3. `Yii::$app->controller->module->getViewPath()` ([:185](https://github.com/yiisoft/yii2/pull/12578/files#diff-64dcde6a6056dd54d40f3d5a0a79b2efR185) will be changed during the runtime for sure
4. Even `$context->getViewPath()` on line [:180](https://github.com/yiisoft/yii2/pull/12578/files#diff-64dcde6a6056dd54d40f3d5a0a79b2efR190) colud be changed as well

IMO, this caching will save some micro-time, but will cause tons of troubles.